Corona-Warn-App/src/main/java/de/rki/coronawarnapp/presencetracing/checkins/common/CheckInExtension.kt

```diff
@@ -14,8 +14,8 @@ val CheckIn.locationName: String
             traceLocationStart.millis > 0 &&
             traceLocationEnd.millis > 0
         ) {
-            val formattedStartDate = traceLocationStart.toUserTimeZone().toString(DateTimeFormat.shortDateTime())
-            val formattedEndDate = traceLocationEnd.toUserTimeZone().toString(DateTimeFormat.shortDateTime())
+            val formattedStartDate = traceLocationStart.toUserTimeZone().toString(DateTimeFormat.shortDate())
```

Comment:
Shouldn't be the two dates removed completely?
![image](https://user-images.githubusercontent.com/22813752/182861395-d8e805e1-277d-4785-ae51-7419275d6716.png)

Reply:
According to the ticket , dates without times should appear , see screenshot in the ticket?
There two kind of events , event that has start and end date and one that does not , I think you choose the last category when you created your event